### DISCUSSION TITLE
Windows Support

### DISCUSSION THREAD

**Discussion by sammy-ette** [sammy-ette/Hilbish#165] [Ideas]
probably long term (or short term) goal since developing on windows is hell due to how slow it is, but windows support would be a good thing

it would bring a more conventional (for unix people) shell, easily cross platform and in a simple scripting language (lua)

the thing though is as i mentioned windows is slow as hell so i cant actually develop on it and windows support itself isnt high priority for me

**Comment by cp9dl** [comment]
> say something!

*something*
